go/ql/lib/semmle/go/security/ExternalAPIs.qll

```diff
@@ -173,6 +181,7 @@ class UnknownExternalApiDataNode extends ExternalApiDataNode {
     // Not a sink for a commonly-used query
     not isACommonSink(this) and
     // Not in a package that has some functions modeled
+    // FIXME
```

Comment:
Can you be more specific about what needs fixing, and what (if anything) is blocking it